### DISCUSSION TITLE
mhy的ai是真的不会玩流萤

### DISCUSSION THREAD

**Discussion by 1492830678** [share121/inter-knot#3224] [其他]
![Image_1724090428204](https://github.com/user-attachments/assets/c7da64cf-a4c2-4923-8ce8-e974767a8cbc)
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Comment by D111Z41SY** [comment]
哈哈哈哈哈昨天拿三月七超击破队打虚构，流萤 同谐主 巡猎三月还有加拉赫，ai三月七会拜师流萤，笑死我了哈哈哈哈哈哈哈哈哈哈哈我想看的就是这个啊！
